pages/index.md

```diff
@@ -3,44 +3,97 @@ name: Introduction
+
+## How to Read This Specification
+
+The specification is divided into several modules.
+Each module describes a mostly self-contained aspect of the DSNP and includes documents detailing the function and intended use the module.
+
+Proposals for some of these future extensions can be found in the [Draft Specifications](#draft-specifications) section and on [GitHub](https://github.com/LibertyDSNP/spec/labels/enhancement),
+however these proposals should be regarded with caution as they are largely incomplete and may be subject to change or deletion without notice.
+
+## Implementation Status
+
+**DISCLAIMER**
+
+The DSNP is still in a work in progress and as such it is currently subject to frequent changes.
+We do not recommend implementing any software based on this document in its current state.
+Once the Protocol Specification reaches an appropriate degree of completeness, we will remove this disclaimer.
+
+### Versions
+
+DSNP Specification Versions follow [Semantic Versioning 2.0](https://semver.org/) for releases.
+Draft Specifications do NOT effect the version number until they are included.
 
-# How to Read This Specification
+| Name | Description | Date |
+| --- | --- | --- |
+| 1.0.0 | Initial Release | In Progress |
 
-The specification is divided into several modules found in the DSNP section of this website.
-Each module describes a self-contained aspect of the DSNP and includes documents detailing the function and intended use the module.
-Generally, each of these documents will include sections on the purpose of the module, assumptions made in the document and the current status of the specification.
+### Draft Specifications
 
-Additionally, the status of each module is also compiled in the [Implementation Status](/Implementation_Status) table for ease of access.
-As further extensions of the protocol are created, more modules may be added to the DSNP directory and status table.
-Proposals for some of these future extensions can be found in the Future Proposals directory under DSNP, however these proposals should be regarded with caution as they are largely incomplete and may be subject to change or deletion without notice.
+| Name | Status | Description |
+| --- | --- | --- |
+| [Archivist](/Draft/Archivists) | DRAFT | Long term DSNP Announcement Storage |
 
-# Specification status definitions
+#### Status Definitions
 
 | Name | Description |
 | --- | --- |
-| **Draft** | Specification is posted as draft PR. It's open for comment and major changes. |
+| **Draft** | Open for comment and major changes. |
 | **Proposed** | Ready for formal review. It should be done except for minor changes. |
 | **Tentative** | This is the accepted plan. The specification should not change unless there are blocking issues. |
-| **Final** | This version will not change. Changes require a new specification process and new specification version number. |
```

Comment:
curios why did we remove this?

Reply:
This is due to moving the non-final stuff into the Draft folder. Once it is out of there it is final, so no need for the status.